sim/pokemon.ts

```diff
@@ -18,6 +18,20 @@ interface MoveSlot {
 	virtual?: boolean;
 }
 
+ /** A Pokemon's move slot as sent to the client. */
+interface RequestMoveSlot {
+	id: string;
+	move: string;
+	pp?: number;
+	maxpp?: number;
+	target?: string;
+	targetZ?: string;
+	disabled?: string | boolean;
+	disabledSource?: string;
+	effectiveness?: number[];
+	effectivenessZ?: number[];
```

Comment:
Needed because of Flying Press and Freeze Dry

Reply:
I think the order is usually `zTarget` and `zEffectiveness`.